### DISCUSSION TITLE
Create3 RPLidar

### DISCUSSION THREAD

**Discussion by dancrissco** [iRobotEducation/create3_docs#372] [Show and tell]
![1680634797345](https://user-images.githubusercontent.com/37241243/229900401-0e312fd0-6905-4177-8fc5-7641ebb61932.jpg)
![1680634796335](https://user-images.githubusercontent.com/37241243/229900460-de405871-8d66-443b-bd71-e0baa0b633ba.jpg)

**Comment by kwujciak** [comment]
This is so cool!

**Comment by slowrunner** [comment]
Is the LIDAR powered via USB from Raspberry Pi4?

What are the "playtimes" with the LIDAR running vs not-spinning (with minimal or no driving)?

Where did you publish your LIDAR attachment plate so others can make/have one made?  (I don't own a 3D-Printer) 

Which GamePad are you using?

Are you running the TurtleBot4 Raspberry Pi image?
